content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,21 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, support is needed in the container runtime and in the OCI container
+runtime.
```

Comment:
Readers are likely to not know what you mean here. Can you clarify?

We might say “CRI-level runtime” and “OCI container runtime”, but then I think we'd do well to add an explanation about the two (maybe to our glossary?)

Reply:
Nah, that seems to complex and we are already using all over the place (not just here) the term "container runtime", I don't think we should change it.

I've adjusted to refer to the "OCI runtime" and added a clarification here too.

Reply:
We've found a place where we need to explain two kinds of container runtimes. You and I know the difference, but plenty of readers will not.

If we don't explain it here, we should expect to be explaining it in GitHub issues (and I'd like to avoid that happening).

Reply:
@sftim What? I've changed it and explained, have you read what I changed? I don't think using "CRI runtime" is worth it, for the things I explained. It is just that.

Reply:
Reviewing on behalf of SIG Docs, I'm disagreeing - I think we should help readers more. Can you convince me / us that we won't have confused readers here?

Reply:
@sftim sure. What do you think is not clear or could cause confusion? It would be great if you share that, to explain.

Let me explain in general, though, what has been already done:

First, I don't think we should use "CRI container runtime". We already use the term "container runtime" and we have been using it for several years, I don't see why we would change that. Furthermore, it is all over the place, it is not easy to change.

Secondly, the OCI runtime is only relevant for documentation in the userns context and maybe even for a few months only. So using a very specific term, instead of changing "container runtime" to "CRI container runtime" seems better.

Third, you asked to clarify and I chose a better name and clarified here what it is already.

What do you think is not clear and could cause confusion?